Assets/MixedRealityToolkit.Services/InputSystem/PointerBehavior.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Specifies how a pointer in MRTK's default input system behaves.
+    /// On: Pointer is always on, regardless of what other pointers are active.
+    /// Off: Pointer is always off, regardless of what other pointers are active.
+    /// Default: Pointer active state is managed by MRTK input system. If it is a near pointer (grab, poke), it
+    /// will be always enabled. If it is not a near pointer, it will get disabled if any near pointer on the 
+    /// same hand is active. This is what allows rays to turn off when a hand is near a grabbable.
+    /// 
+    /// By default, pointer behaviors are set to default state.
+    /// </summary>
+    public enum PointerBehavior
+    {
+        /// <summary>
+        /// Pointer is always on, regardless of what other pointers are active.
```

Comment:
This seems troublesome. I feel like a lot of developers are just going to set On/Off but really most will want the scenario of Off vs Default. Having a pointer On seems dangerous because we have all of that custom logic in PointerMediator to do special steps we think are good

I'm not exactly sure how I would fix this. Maybe at least make Default = 0 since that is the default we want

Reply:
Following up on this, I know it's annoying, but it might also be good to change PointerOnOff example scene content to be like PointerBehaviorExample instead. Again the problem is people will generally not want On/off but Default/Off

Reply:
Also now seeing the PR and understanding everything, I was an idiot before. These should be AlwaysOn and AlwaysOff enums. Sorry about that

Reply:
Updated to the following enum:

AlwaysOn,
AlwaysOff,
Default